### ISSUE TITLE
justify-items autoprefixing broken

### ISSUE THREAD

**Issue by codepunkt** [thysultan/stylis.js#68]
The result of

```javascript
stylis('#wat', `justify-items: center;`))
```

is this, which confuses me - i've never seen those properties. Is this correct?
```css
#wat {
  -webkit-box-packjustify-items: center;
  -webkit-justify-items: center;
  -ms-flex-packjustify-items: center;
  justify-items: center;
}
```

**Comment by morajabi** [comment]
No, I'm pretty sure it's a bug.

**Comment by thysultan** [comment]
Patched in 3.4.0
